saleor/csv/events.py

```diff
@@ -0,0 +1,41 @@
+from typing import TYPE_CHECKING
+
+from ..account.models import User
+from . import ExportEvents
+from .models import ExportEvent
+
+if TYPE_CHECKING:
+    from .models import ExportFile
+
+
+def export_event(*, export_file: "ExportFile", user: User):
```

Comment:
One last change - I would rename this function to `export_started_event` - it will be more consistent with other functions in this module. Now it sounds like this function was used to export some events which is not exactly true :)